examples/src/http_server_html_template/src/build.rs

```diff
@@ -0,0 +1,3 @@
+pub fn main() {
+    let _ = pluto_templating::initialize_templating_engine("views");
```

Comment:
we shouldnt use _ in here try to handdle error in someway or unwrap it if it realy should drop one